### PR TITLE
Allow for serving of a static site via nginx if docroot is set. Remove proxy rules from redirects.

### PR DESCRIPTION
**Pull request by cafuego**
Hey mig, if you have a spare few mins, could you review this? No urgency... I'll be off for the next 20  hours anyway.

I think I need this for a BNM site, would like to bypass Apache because really... it's just static HTML. Pretty sure this should be safe for that. Also removing extraneous config from the redirect vhosts (they only return a 301 anyway) and fixing ownership of the htauth file. No nginx user on Ubuntu.

This change is for the tv4me-ph-static site (see channelplay staging yaml in puppet-server) for which I've currently done a manual change on the nginx vhost.

### CODE REVIEW

File: manifests/non_ssl.pp [hunk 1]
```diff
@@ -10,6 +10,7 @@
   $auth_user     = "",
   $auth_pass     = "",
   $proxy_timeout = undef,
+  $docroot       = undef.
```
Comment (mig5) [inline manifests/non_ssl.pp]: don't think there should be a full stop here

File: manifests/non_ssl.pp [hunk 2]
```diff
@@ -35,8 +36,8 @@
         file      => "/etc/nginx/auth/${auth_user}",
         password  => $auth_pass,
         realm     => 'realm',
-        owner     => root,
-        group     => nginx,
+        owner     => 'root',
+        group     => 'www-data',
```
Comment (mig5) [inline manifests/non_ssl.pp]: should the perms be changed on the earlier /etc/nginx/auth File definition (directory) as well, then?

File: manifests/non_ssl.pp (hunk 1 above)
Comment (cafuego) [inline manifests/non_ssl.pp]: correct

File: manifests/non_ssl.pp (hunk 2 above)
Comment (cafuego) [inline manifests/non_ssl.pp]: much of a muchness i think, mode/owner allows for full read anyway

**Comment by cafuego** [comment]
... client changed their mind, I think. So we may not need this anymore now.

**Comment by cafuego** [comment]
Site is now defunct, drop PR.

**Pull request closed without merging**